pkg/graph/schema.resolvers_accessibility_request_test.go

```diff
@@ -126,7 +126,7 @@ func (s *GraphQLTestSuite) TestAccessibilityRequestQuery() {
 					authorName
 				}
 			}
-		}`, accessibilityRequest.ID), &resp, testhelpers.AddAuthWithAllJobCodesToGraphQLClientTest("TEST"))
+		}`, accessibilityRequest.ID), &resp, addAuthWithAllJobCodesToGraphQLClientTest("TEST"))
```

Comment:
Just an FYI we'll be deleting all the accessibility request code at some point in the near future.

Reply:
that's great news, thank you for the heads up!